README.md

```diff
@@ -81,19 +81,29 @@ issue log after verifying that the issue does not already exist.
 Comments on existing issues are also welcome.
 
 # Installation
+Preferred method of installation is `pip install .` 
+(or `pip install git+https://github.com/HERA-Team/linsolve`). This will install all
+dependencies. See below for manual management of dependencies.
+
 ## Dependencies
-First install dependencies. 
+If you use `conda` and would like to ensure that dependencies are installed with `conda`
+rather than `pip`, you should execute::
+
+    $ conda install "numpy>=1.10" scipy
 
-* numpy >= 1.10
-* scipy
 
-## Install linsolve
-Clone the repository using
-```git clone https://github.com/HERA-Team/linsolve.git```
+## Development
```

Comment:
Do we really need this? Does anyone do this?

Reply:
Sorry, I'm not sure what you're pointing to here? People definitely want to `conda` install packages before installation. People also develop `linsolve`, and I for one, if developing, *always* use a fresh environment.

Reply:
I don't personally do this, I don't think @AaronParsons does either. I'd generally prefer not to add boilerplate to the README that distracts from the main point of the document.

Reply:
You don't use a fresh environment? I would argue that just because two developers don't do it this way doesn't mean that others don't, and neither does it mean that you *shouldn't*.

Reply:
Nope. I guess I don't feel too strongly on this point if we're trying to standardize as a collaboration...

Reply:
I don't personally do this and it's usually not necessary. I'd generally prefer not to add verbiage to the README that distracts from the main point of the document.